localflavor/mx/mx_states.py

```diff
@@ -9,11 +9,11 @@
     ('BCN', _('Baja California')),
     ('BCS', _('Baja California Sur')),
     ('CAM', _('Campeche')),
+    ('CDMX', _('Ciudad de México')),
```

Comment:
@jackboot7 the official abbreviation is CDMX, so it would probably need the state field to be updated.